ui/jest.setup.ts

```diff
@@ -0,0 +1,27 @@
+import { chromium, Browser, Page } from "playwright";
+
+let browser: Browser;
+let page: Page;
+
+const username = process.env.APP_USERNAME ?? "";
+const password = process.env.APP_PASSWORD ?? "";
+
+beforeAll(async () => {
+  browser = await chromium.launch();
+  page = await browser.newPage();
+  await page.goto(
+    "https://console.amq-streams-ui.us-east.containers.appdomain.cloud/home"
```

Comment:
this should be coming from an env variable as well, end everywhere where the domain is used should use the env variable as well